pvanalytics/util/_fit.py

```diff
@@ -1,26 +1,17 @@
 """Internal module for curve fitting functions."""
 import numpy as np
 import scipy.optimize
-import pandas as pd
 
 
-def _to_minute_of_day(index):
-    # Transform the index into minutes of the day. If `index` is a
-    # DatetimeIndex then it is converted to an Int64Index with values
-    # equal to the minute of the day since midnight. Any other type
-    # and a ValueError is raised.
-    if isinstance(index, pd.DatetimeIndex):
-        return index.hour * 60 + index.minute
-    raise ValueError("cannot convert index to minutes since midnight")
-
-
-def quadratic(data):
+def quadratic(x, y):
     """Fit a quadratic to the data.
 
     Parameters
     ----------
-    data : Series
-        Series of power or irradiance measurements.
+    x : Series
+        x values for data in `y`.
```

Comment:
Since `quadratic` and `quartic_restricted` are exposed we should provide an example of `x` that connects to the anticipated use. Is `x` a DatetimeIndex, minute of the day, or ?

Reply:
I'll add an example of `x` as minute of the day. This is currently the only way it is used.